platform/o.n.bootstrap/launcher/windows/version.h

```diff
@@ -19,9 +19,9 @@
 
 #define COMPANY ""
 #define COMPONENT "Apache NetBeans Platform Launcher"
-#define VER "101.1.0.0"
-#define FVER 101,1,0,0
-#define BUILD_ID "101100"
+#define VER "101.1.1.0"
+#define FVER 101,1,1,0
+#define BUILD_ID "101110"
```

Comment:
i bumped the version in both, IDE and platform launcher `version.h` - just to be sure. (Its all in the same zip artifact anyway)

Reply:
Also need the `.manifest` files.

Reply:
split into extra commit and updated the manifests